### PR TITLE
.github/test.yml: Only test for supported Go versions

### PR DESCRIPTION
**Pull request by HaraldNordgren**

### CODE REVIEW

**Comment by HaraldNordgren** [comment]
Ping @mhemmings

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -4,10 +4,10 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.13.x, 1.14.x, 1.15.x]
+        go-version: [oldstable, stable]
```
Comment (HaraldNordgren) [inline .github/workflows/test.yml:10]: These aliases now exist: https://github.com/actions/setup-go?tab=readme-ov-file#using-stableoldstable-aliases

Comment (HaraldNordgren) [reply .github/workflows/test.yml]: Ignore that, it seems it didn’t work for use these aliases. I reverted it.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -4,15 +4,15 @@ on: [push, pull_request]
...
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: Setup Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v5
```
Comment (HaraldNordgren) [inline .github/workflows/test.yml:15]: This version change is required to make the stable alias work.

**Review by mhemmings — APPROVED** [review]

**Pull request merged**